src/spatial_reaction_systems/lattice_reaction_systems.jl

```diff
@@ -43,7 +43,7 @@ u0 = [:X => rand(5,5)]
 tspan = (0.0, 1.0)
 ps = [:p => 1.0, :d => 0.5, :D => 0.1]
 oprob = ODEProblem(lrs, u0, tspan, ps)
-osol = solve(oprob)
+osol = solve(oprob, Tsit5())
```

Comment:
Are you sure Tsit5 is a good recommendation for a spatial system? I would think you'd want an implicit solver often?